modules/win_packages/manifests/win_exe_pkg.pp

```diff
@@ -18,6 +18,7 @@
     exec { "${title}install":
         command => "${pkgdir}\\${pkg} ${install_options_string}",
         creates => $creates,
+        timeout => 600,
```

Comment:
For the future we should identify which packages take an excessive amount of time to install, and see if there is any way to cut that down.

Reply:
This was due to win10 sdk. Upgrading that will hopefully resolve any timeouts.